app/assets/javascripts/client/views/reset_password/request_reset.html.erb

```diff
@@ -1,64 +1,55 @@
 <div class="login signup">
   <div class="row secondary">
-    <div class="container">
-      <div class="col-md-4">
-        <h1>Forgot your password?</h1>
-        <h3 class="subheading" >We'll email you instructions to reset your password.</h3>
-        <p class="resetpass-help-text">Need help? Contact us at support@mail.pdredesign.org.</p>
-      </div>
+    <div class="col-md-4">
+      <h1>Forgot your password?</h1>
+      <h3 class="subheading" >We'll email you instructions to reset your password.</h3>
+      <p class="resetpass-help-text">Need help? Contact us at support@mail.pdredesign.org.</p>
+    </div>
 
-      <div class="col-md-8 form">
-        <div class="row">
-          <div class="col-md-7 col-sm-6">
-            <alert ng-repeat="alert in alerts" type="{{alert.type}}" close="closeAlert($index)">{{alert.msg}}</alert>
-          </div>
+    <div class="col-md-8">
+      <div class="row">
+        <div class="col-md-12">
+          <alert ng-repeat="alert in alerts" type="{{alert.type}}" close="closeAlert($index)">{{alert.msg}}</alert>
         </div>
+      </div>
 
-        <form id="resetForm">
-        <div class="row">
-          <div class="col-md-7 col-sm-6">
-            <input
-              required
-              ng-model="email"
-              class="form-group form-control"
-              placeholder="Email Address*"
-              type="text">
-            <input
-              ng-disabled="resetForm.$invalid"
-              type="button"
-              ng-click="requestReset(email)"
-              class="btn btn-devise btn-danger pull-right"
-              name="commit"
-              value="Request Reset Email">
-          </div>
+      <form id="resetForm" class="form" ng-submit="requestReset(email)">
+        <div class="form-group">
+          <input
+            required
+            ng-model="email"
+            class="form-control"
+            placeholder="Email Address*"
+            type="email">
+          <input
+            ng-disabled="resetForm.$invalid"
+            type="submit"
+            class="btn btn-devise btn-danger pull-right"
```

Comment:
I wonder what this `btn-devise` class represents.  There's no evidence if of it in the existing codebase.